README.md

```diff
@@ -27,6 +27,7 @@ A list of resources connected to the [SpaceTraders API](https://spacetraders.io/
 * [space_trader](https://github.com/HOWZ1T/space_trader) - An Golang api wrapper with internal caching and an event system.
 * [SpaceMongers](https://github.com/ericgroom/space_mongers) - Elixir API wrapper with integrated rate limiting.
 * [spacetraders-sdk](https://github.com/notVitaliy/spacetraders-io) - A Javascript/Typescript SDK.
+* [spacetraders-php](https://github.com/rayblair06/SpaceTradersPHP) - A PHP Api Wrapper.
```

Comment:
The name of the project should be the name of the project. Since your repository is named `SpaceTradersPHP` and the package which is published is named `rayblair/spacetradersphp` I think one of
- `spacetradersphp`
- `SpaceTradersPHP`

should be used.

```suggestion
* [spacetradersphp](https://github.com/rayblair06/SpaceTradersPHP) - A PHP Api Wrapper.
```